openhands_ext/test_storage.py

```
@@ -0,0 +1,203 @@
+        # Load current settings
+        current_settings = storage_manager.load_settings(user_id)
+        
+        # Update with new data
+        for key, value in settings_data.items():
+            if hasattr(current_settings, key) and key != 'user_id':
+                setattr(current_settings, key, value)
+        
+        # Save updated settings
+        storage_manager.save_settings(user_id, current_settings)
+        
+        return {"message": "Settings updated successfully"}
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)